txnsync/incoming.go

```diff
@@ -32,6 +34,68 @@ type incomingMessage struct {
 	encodedSize    int
 }
 
+// incomingMessageQueue manages the global incoming message queue across all the incoming peers.
+type incomingMessageQueue struct {
+	incomingMessages chan incomingMessage
+	enqueuedPeers    map[*Peer]struct{}
```

Comment:
do the values of enqueuedPeers ever get called? Not sure but is putting the value of the map as a struct{} the right way to go?

Reply:
I couldn't find any particular value that I would need in this map.
In particular, this is very similar to a `map[*Peer]bool` where in order to test existence, one could just `if enqueuedPeers[peer] { ... }` since the default value for bool is false.

But I don't have any strong feeling toward either.